File: concept/03_loops/all/all_loops.cpp
#include <iostream>
using namespace std;
 
int main() {
     
    // for loop Ex.
    cout << "For loop : ";
    for(int i = 0; i < 5; i++) {
        cout << i << " ";
    }

    cout << endl;
     
    // While loop Ex.
    cout << "While loop : ";
    int j = 0;
    while(j < 5) {
        cout << j << " ";
        j++;
    }
     
    cout << endl;
    // do-while loop Ex.
    cout << "Do-while loop : ";
    int k = 0;
    do {
        cout << k << " ";
        k++;
    } while(k < 5);
     
    return 0;
}

File: concept/03_loops/all/all_loops.dart
import 'dart:io';

void main() {
  // for loop Ex.
  stdout.write("For loop : ");
  for (int i = 0; i < 5; i++) {
    stdout.write(i.toString() + " ");
  }

  // While loop Ex.
  stdout.write("\nWhile loop : ");

  int j = 0;
  while (j < 5) {
    stdout.write(j.toString() + " ");
    j++;
  }

  // do-while loop Ex.
  stdout.write("\nDo-while loop : ");
  int k = 0;
  do {
    stdout.write(k.toString() + " ");
    k++;
  } while (k < 5);
}
